### PR TITLE
add-eks-templates

### PR DESCRIPTION
**Pull request by calvix**
add templates to create a EKS cluster
towards https://github.com/giantswarm/roadmap/issues/2609

### CODE REVIEW

**Comment by calvix** [comment]
this is just very alpha version of EKS cluster, not meant for general use yet

File: helm/cluster-eks/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: cluster-eks
+home: https://github.com/giantswarm/cluster-eks
+sources:
+  - https://github.com/giantswarm/cluster-eks
+description: A helm chart for creating Cluster API EKS clusters with the AWS infrastructure provider (CAPA).
+icon: https://s.giantswarm.io/app-icons/aws/1/dark.svg
+type: application
+version: 0.1.0
+annotations:
+  application.giantswarm.io/team: "hydra"
```
Comment (fiunchinho) [inline helm/cluster-eks/Chart.yaml:11]:
```suggestion
  application.giantswarm.io/team: "phoenix"
```

**Review by fiunchinho — CHANGES_REQUESTED** [review]
Update the team, otherwise LGTM

File: helm/cluster-eks/Chart.yaml (hunk 1 above)
Comment (calvix) [reply helm/cluster-eks/Chart.yaml]: fixed, It was actually copied from `cluster-aws` so i created a issue to fix it there as well

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**